### ISSUE TITLE
Cut a new release?

### ISSUE THREAD

**Issue by voxpelli** [caolan/forms#222]
I think there are enough changes to merit a new release to be cut?

**Comment by ljharb** [comment]
The only change would be your 11200d84278d269b6727abe021861f3cc1e02bff commit - everything else isn't material.

**Comment by voxpelli** [comment]
Right, a quick look showed quite a few more commits, but maybe they were all internal then.

That change of mine is indeed the one I was looking at making use of though.
